src/gui/macOS/ui/FileProviderSyncStatus.qml

```diff
@@ -71,7 +71,7 @@ GridLayout {
         text: qsTr("Request sync")
         visible: !root.syncStatus.syncing
         hoverEnabled: true
-        onClicked: root.controller.signalFileProviderDomain(root.accountUserIdAtHost)
+        onClicked: root.controller.domainSignalRequested()
```

Comment:
there is no `controller` property in `root`

```suggestion
        onClicked: root.domainSignalRequested()
```

Reply:
Thanks, I should have noticed this